packages/stream_chat_flutter/example/pubspec.yaml

```diff
@@ -27,9 +27,9 @@ dependencies:
   cupertino_icons: ^1.0.3
   flutter:
     sdk: flutter
-  stream_chat_flutter: ^2.2.1
-  stream_chat_localizations: ^1.1.0
-  stream_chat_persistence: ^2.2.0
+  stream_chat_flutter: ^3.4.0
+  stream_chat_localizations: ^2.1.0
+  stream_chat_persistence: ^3.1.0
```

Comment:
@sachaarbonel isn't it better to use the local version from path?

Reply:
isn't really necessary as we run them after `melos bs` which does the same thing.

Reply:
yeah but then once we push a new release we need to update this as well and it's very easy to forget

Reply:
fine then 👍🏼

Reply:
@sachaarbonel can you update this?